src/lib/keys.js

```diff
@@ -53,4 +53,6 @@ export function allKeys( arg ) {
   return arg ? (arg.constructor === Symbol || typeof arg === 'symbol') : Symbol( 'allKeys' );
 }
 
+export const ALL_KEYS = Symbol('ALL_KEYS')
```

Comment:
The reason it confuses me is, that `allKeys` function currently returns a new `Symbol` instance on every call instead of re-using same instance. Any reason for that!